astro/src/content/blog/fusionauth-re-invent-2023.mdx

```diff
@@ -0,0 +1,59 @@
+excerpt_separator: "{/* more */}"
+---
+
+AWS re:Invent is one of the largest technical conferences in the world. In the cloud space, it is the go-to event. This week there will be tens of thousands of attendees in Las Vegas, Nevada. The [FusionAuth team will be among them](/aws-reinvent23), and we'd love to see you there!
+
+{/* more */}
+
+## Why the team is attending
+
+We're going to re:Invent for many reasons.
+
+One is that FusionAuth and our clients often deploy on the AWS public cloud. Some clients use vanilla EC2 instances with RDS, others deploy on solutions like EKS and Fargate (yes, [FusionAuth supports Kubernetes](/docs/v1/tech/installation-guide/kubernetes/fusionauth-deployment)).
+
+AWS re:Invent is an opportunity to meet developers where they are both during talks and in the ever-present hallway track. We look forward to exchanging knowledge and ideas with folks building on the AWS platform.
+
+We're also excited to chat with attendees about authentication. We like to hear from developers about what they are building on top of solutions like FusionAuth, Auth0 and Cognito.
+
+You can migrate from [Auth0](/docs/v1/tech/migration-guide/auth0) to FusionAuth and we're of course happy to help. Migrating away from Cognito isn't as straightforward, but [there are ways](/docs/v1/tech/migration-guide/cognito).
+
+We're also looking forward to some AWS re:Invent sessions. 
+
+Exciting ones include 
+- SEC209 | Modernize authorization: Lessons from cryptography and authentication
+- DAT333 | Building highly resilient applications with Amazon DynamoDB
+- SEC322 | Integrate apps with Amazon Cognito and Amazon Verified Permissions
+- BOA209 | Build verifiable and effective application authorization in 40 minutes
+
+Of course, it isn't all education. There's also fun to be had!
+
+One fun thing to do is join [Corey Quinn's annual nature walk](https://www.lastweekinaws.com/blog/aws-re-invent-what-you-actually-need-to-know-before-you-go/) where he skewers Expo vendors. This is happening on Wednesday at 1pm.
+
+There are also numerous parties. If you are attending AWS, chances are at least one vendor has emailed you with an invite. This Twitter account, [@reInventParties](https://twitter.com/reInventParties/), is a great way to discover additional opportunities for fun.
+
+Finally, if nature is your thing, we've heard [Red Rock Canyon](https://www.blm.gov/programs/national-conservation-lands/nevada/red-rock-canyon) has good hiking and climbing.
+
+## Person On The Street
+
+Have you ever wanted to be one of those people in Vegas that they walk up to and ask questions to win prizes? Well, you are in luck because from the moment my shoes hit the ground at re:Invent this year I will be recording all the action. I will be asking some hard-to-answer questions about Authentication, but when you get the answer right you will win a collector's set of FusionAuth trading cards. With each set, there will be an opportunity to win swag at our booth #152.
```

Comment:
"With each set, there will be an opportunity to win swag at our booth #152."

The way I read this is that you use the cards for a chance to win something?  Is that right or do you just get something for the cards?